refgenie/asset_build_packages.py

```diff
@@ -72,6 +72,32 @@
             "cut -f 1,2 {asset_outfolder}/{genome}.fa.fai > {asset_outfolder}/{genome}.chrom.sizes",
         ]
     },
+    "fasta_txome_extract": {
+        DESC: "cDNA sequences in FASTA format exracted from the genome and annotation files, indexed FASTA (produced with samtools index) and chromosome sizes file",
+        ASSETS: {
+            "fasta_txome": "{genome}.fa",
+        },
+        REQ_FILES: [],
+        REQ_ASSETS: [
+            {
+                KEY: "fasta",
+                DEFAULT: "fasta",
+                DESC: "fasta asset for genome"
+            },
+            {
+                KEY: "gtf",
+                DEFAULT: "gencode_gtf",
+                DESC: "GTF file for exonic features extraction"
```

Comment:
since the required input here is of asset class, the "GTF file" in the description might be confusing